source/code/include/playfab/PlayFabEvent.h

```diff
@@ -16,8 +16,8 @@ namespace PlayFab
     enum class PlayFabEventType
     {
         Default, // Default type (e.g. the one set by global configuration)
-        Lightweight, // Event is meant to bypass processing on the PlayFab server side and be sent instead directly to One Collector (1DS).
-        Heavyweight // Event is meant to be sent to the PlayFab server.
+        Lightweight, // WriteTelemetryEvents
```

Comment:
>WriteTelemetryEvents [](start = 24, length = 20)

I agree that for someone like us it's pretty obvious what these comments mean :) For someone else though, who just picked up our SDK it might be more useful if we add a few more words here. For example, "Event will be sent using WriteTelemetryEvents method in EventsAPI"